x11/makedepend/Portfile

```diff
@@ -1,7 +1,7 @@
 PortSystem          1.0
 
 name                makedepend
-version             1.0.5
+version             1.0.6
 revision            1
```

Comment:
The `revision` needs to be set back to 0 when increasing the `version`.

Reply:
My bad, thanks for the check. Done.